modules/rest_api/controllers/services/rest.php

```diff
@@ -869,7 +924,7 @@ private function getEsPostData($postObj, $format, $file) {
           $fields[] = 'event.date_start';
           $fields[] = 'event.date_end';
         }
-        elseif ($field === '#lat_lon#' || $field === '#lat#' || $field === '#lon#') {
+        elseif ($field === '#lat_lon#' || preg_match('/^#(lat|lon):(.*)#$/', $field)) {
```

Comment:
Doesn't this regex require the : to be present? Existing #lat# or #long# implementations would fail rather than use the default?

Reply:
Sorry yes - I overlooked that and literally just come across it when testing the idc.jquery special handlers. I'll fix it.